test/src/cli/flutter_cli_test.dart

```diff
@@ -381,6 +381,11 @@ void main() {
         verify(
           () => logger.write(any(that: contains('Some tests failed.'))),
         ).called(1);
+        verify(
+          () => logger.err(
+            any(that: contains('- [FAILED] test/example_test.dart 4:5')),
```

Comment:
The output here, can we somehow turn that into:
```
[FAILED] test/example_test.dart:4:5
```

Most IDEs know that this means a certain file from the current root at given column and line. This would allow developers to immediately jump to the correct line.

Reply:
Sure 👍

Reply:
Done